src/main/java/com/cmsc495/ticketsystem/service/TicketService.java

```diff
@@ -23,26 +23,52 @@ public class TicketService {
     @Autowired
     private TicketRepository ticketRepository;
 
-    public void changeTicketAndStatus(Long ticketId, String status, String noteContent) {
-        TicketModel ticket = ticketRepository.findById(ticketId).orElseThrow(()
-                -> new RuntimeException("Ticket not found"));
+    /**
```

Comment:
Im not opposed to the addition of these comments, but the formatting looks a bit off because it's not consistent with the rest of the project.